app/controllers/api/stories_controller.rb

```diff
@@ -17,6 +17,22 @@ def after_create_resource(res)
     res.create_story_distributions
   end
 
+  def after_update_resource(res)
+    dist_tmpl = res
+      .try(:series)
+      .try(:distributions)
+      .try(:find) { |d| d.kind == "podcast" }
+      .try(:audio_version_template_id)
+
+    version_tmpl = res.try(:audio_versions)
+      .try(:find) { |av| !av.audio_version_template_id.nil? }
+      .try(:audio_version_template_id)
+
+    if dist_tmpl && version_tmpl && dist_tmpl == version_tmpl && res.distributions.empty?
+      res.create_story_distributions
```

Comment:
this method is going to make distros for all distributions that are and are not podcasts - you've prevented it making a dupe podcast episode distro above, but would still create other dupe distros for any other kind of series distro